code/modules/clothing/glasses/hud.dm

```diff
@@ -103,6 +103,20 @@
 		if(report)
 			report.read_paper(user)
 
+/obj/item/clothing/glasses/hud/health/science
+	name = "\improper custom HealthMate HUD" // combined HealthMateHUD and Reagent Scanner HUD for CMO
```

Comment:
```suggestion
	name = "custom HealthMate HUD" // combined HealthMateHUD and Reagent Scanner HUD for CMO
```
improper is only needed for names that you're beginning with a capital letter, as it overrides it being treated as a proper/improper noun